test/watcher.js

```diff
@@ -278,4 +278,29 @@ describe('watcher', function() {
     output = await run(bundle);
     assert.equal(output(), 10);
   });
+
+  // For some reason watch isn't triggering an onChange event on this test
+  it.skip('should rebuild if a file gets added to a glob path', async function() {
```

Comment:
Not sure why this test isn't working, it's working like a charm in a test project